server/src/DbtTextDocument.ts

```diff
@@ -331,8 +333,8 @@ export class DbtTextDocument {
   }
 
   onSignatureHelp(params: SignatureHelpParams): SignatureHelp | undefined {
-    const text = this.rawDocument.getText(this.getTextRangeBeforeBracket(params.position));
-    return this.signatureHelpProvider.onSignatureHelp(params, text);
+    const text = this.rawDocument.getText(getTextRangeBeforeBracket(this.rawDocument.getText(), params.position));
```

Comment:
Why do we need `getTextRangeBeforeBracket`? It seems that we can use `getWordRangeAtPosition`.

Reply:
Now I use `getWordRangeAtPosition` internally in https://github.com/fivetran/dbt-language-server/pull/99/files#diff-7ee293d1f12e6e9036c6bf8d2558a887989691cf80f7f394c4d61dca27594e22R206. But in this method we need find Bracket depending on context. For `foo(bar(abc), d) ` this methos should return foo if cursor located in position of `d`. I'm not sure that we can solve this only by using `getWordRangeAtPosition`, or we should have very complex regexp for it.

Reply:
Seems reasonable.